package/Dockerfile.lighthouse-coredns

```diff
@@ -0,0 +1,9 @@
+FROM debian:stable-slim
```

Comment:
Would it be possible to use ubi8 here to be in line with the rest of the images we build?

Reply:
yeah we already discussed it I will open another PR for that